BlogWebsiteMVC/View/login_view.php

```diff
@@ -1,3 +1,4 @@
+<!--
```

Comment:
Why you have commented this code?
If this code is not required then please remove the same.